Pixel Game/Assets/Scripts/ShopScreen.cs

```diff
@@ -32,42 +18,24 @@ void Start()
                 item.SetAlphaOfColor(0f);
             }
         }
+        gameObject.GetComponent<CanvasGroup>().blocksRaycasts = true;

```

Comment:
Is this to disable the ability to click on shopscreen when closed? (blocks raycast)

Reply:
No, this was needed to make it possible to click on the shopscreen when open. I'm not sure how this works tbh.

Reply:
We can look into it together. I need to learn more of it aswell.